app/components/works/add_files_component.html.erb

```diff
@@ -78,36 +78,36 @@
     <div data-file-uploads-target="globusUpload">
       <p>Please notify us at <a href="mailto:sdr-contact@lists.stanford.edu">sdr-contact@lists.stanford.edu</a> if you are uploading 1TB or more of content.</p>
       <div class="container py-4">
-        <div class="row g-3">
-          <div class="col">
+        <ul class="row g-3 list-unstyled"">
```

Comment:
Looks like there may be an extra `"` character here

Reply:
I'm wondering about using `ul` here, as that implies a lack of order in the list. Is `ol` more semantically appropriate? Or is there a different HTML tag that might be more appropriate for this content? (If so, I'm not sure what that might be.)

Reply:
Oops, good catch.